NEWS.md

```diff
@@ -1,5 +1,6 @@
 # mlr3tuning 0.14.0.9000
 
+* feat: Add dictionary `mlr_callbacks` with predefined callbacks and `callback_tuning()` function to create custom callbacks.
```

Comment:
I would rather say that we populate `mlr_callbacks` with callbacks for `mlr3tuning` as the dictionary is added in `mlr3mis`